dallinger/docker/tools.py

```diff
@@ -317,14 +326,15 @@ def build_image(
             ./prepare_docker_image.sh
         # We rely on the already installed dallinger: the docker image tag has been chosen
         # based on the contents of this file. This makes sure dallinger stays installed from
-        # /dallinger, and that it doesn't waste space with two copies in two different layers.
-
+        # /dallinger, and that it doesn't waste space with two copies in two different layers.\
```

Comment:
Not sure why you added that backslash here. Everything else LGTM.

Reply:
Ah thanks for spotting!

Reply:
Ah thanks for spotting!